README.md

```diff
@@ -22,8 +22,10 @@
 
 # Learn about CLAP
 
-CLAP stands for **CL**ever **A**udio **P**lugin.
-It is an audio plugin ABI which defines a standard for *Digital Audio Workstations* and audio plugins (synthesizers, audio effects, ...) to work together.
+CLAP stands for **CL**ever **A**udio **P**lugin. It is an *API* that provides a
+stable binary interface with backwards compatibility to define a standard for
```

Comment:
So to clear things up and not shoot ourselves in the foot in the future: CLAP provides binary compatibility across _all major and minor versions_, but doesn't necessarily provide source compat across releases. I have a slight reservation about the term 'generates' because we don't generate binary compatibility; it is a result of careful design decisions within this library.

Reply:
I always think about the initial descriptions of projects as "How can I describe it with the least words possible for people to grasp what it's about". I think the explanation of `ext` and `ext/draft` should be mentioned below in their specific sections. How about:

CLAP stands for **CL**ever **A**udio **P**lugin. It is an interface that
provides a stable ABI to define a standard for *Digital Audio Workstations* and
audio plugins (synthesizers, audio effects, ...) to work together.

The ABI, or **A**pplication **B**inary **I**nterface, serves as a means of
communication between a host and a plugin. It provides backward compatibility,
that is, a host that uses a specific version of CLAP X.x can load a plugin with
an older version of CLAP Y.y.

Perhaps this is clear and quick to understand?

Reply:
Yeah just clap x.x should be “a plugin binary compiled with clap 1.x can be loaded by any other clap 1.x”

Reply:
Yeah, sounds good to me :)

Reply:
I understand that you're both happy with the change.
I merge it do dev :)